apps/mobile/src/features/Staking/Governance/common/Action/Action.tsx

```diff
@@ -48,6 +48,7 @@ export const Action = ({
             a.relative,
             a.rounded_sm,
             !showGradient && [a.border, {borderColor: p.gray_200}],
+            showGradient && {backgroundColor: p.bg_color_min},
```

Comment:
i think need flat it or invert the login to be within
```suggestion
            showGradient && [ta.bg_color_min],
```